File: cassandra/scala/update/cassandra_update.scala
// -------------------------------------------------------------- 
//
//	cassandra_update.scala
//
//						Sep/05/2013
//
// -------------------------------------------------------------- 
import	java.sql.Connection
import	java.sql.DriverManager

// -------------------------------------------------------------- 
object cassandra_update
{
// -------------------------------------------------------------- 
def main (args: Array[String])
{
	println ("*** 開始 ***")

	val key = args(0)
	val population_in = args(1).toInt

	val str_connect = "jdbc:cassandra://localhost:9160/city"

	var conn = DriverManager.getConnection (str_connect)

	cassandra_manipulate.cassandra_update_proc (conn,key,population_in)

	conn.close ()

	println ("*** 終了 ***")

}

// -------------------------------------------------------------- 
}
// -------------------------------------------------------------- 

File: cassandra/java/update/cassandra_update.java
// -------------------------------------------------------------- 
//
//	update/cassandra_update.java
//
//					Sep/05/2013
//
// -------------------------------------------------------------- 
import	java.sql.Connection;
import	java.sql.DriverManager;

// -------------------------------------------------------------- 
public class cassandra_update
{
// -------------------------------------------------------------- 
public static void main(String[] args) throws Exception
{
	System.out.println ("*** 開始 ***");

	String	key = args[0];
	int	population = Integer.parseInt (args[1]);
	System.out.print ("\tkey = " + key);
	System.out.println ("\tpopulation = " + population);

	String str_connect = "jdbc:cassandra://localhost:9160/city";

	Connection conn = DriverManager.getConnection (str_connect);

	cassandra_manipulate.cassandra_update_proc (conn,key,population);

	conn.close ();

	System.out.println ("*** 終了 ***");
}

// -------------------------------------------------------------- 
}
// -------------------------------------------------------------- 
